src/robot_interface/telemetry/payloads.py

```diff
@@ -16,3 +19,25 @@ class TelemetryBatteryPayload:
     battery_level: float
     robot_id: str
     timestamp: datetime
+
+
+@dataclass
+class VideoStream:
+    name: str
+    url: str
+
+
+@dataclass
+class AvailabilityPayload:
+    robot_name: str
+    robot_model: str
+    serial_number: str
+    video_streams: List[VideoStream]
+    host: str
+    port: int
+    robot_available: bool
+    isar_available: bool
+    current_state: State
+    current_mission_id: str
+    current_task_id: UUID
+    current_step_id: UUID
```

Comment:
Some of these properties might be moved to get information endpoint so we dont publish alot of unnecessary info